guides/common/modules/proc_increasing-the-logging-level-for-foreman.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="increasing-the-logging-level-for-foreman"]
+= Increasing the logging level for Foreman
+
+You can find the log in `/var/log/foreman/production.log`.
```

Comment:
Unless you changed the logging to use journald: https://docs.theforeman.org/nightly/Administering_Project/index-katello.html#configuring-logging-type-and-layout_admin